packages/contracts/utils/environment.ts

```diff
@@ -0,0 +1,58 @@
+import {HARDHAT_NETWORK_NAME} from 'hardhat/plugins';
+import {Network} from 'hardhat/types';
+
+/**
+ * Provides default values for environment variables if running against a hardhat node
+ * else requires that the environment variables are set
+ * @param network the hardhat network object
+ * @param name of the environment variable
+ * @param defaultValue the fallback value to be used if not set and in development mode
+ * @returns the value of the environment variable, or a fallback if possible
```

Comment:
```suggestion
 * @param network The hardhat network object.
 * @param name The name of the environment variable.
 * @param defaultValue The fallback value to be used if not set and in development mode.
 * @returns The value of the environment variable, or a fallback if possible.
```
For the contracts, we use full sentences. I am not sure if we follow this practice consequently in other places, but I think it would be good. 
Feel free to ignore this comment